src/components/wrappers/FeatureWrapper/index.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ReactElement } from 'react'
+
+import { Navigate } from '@/components/common/Navigate'
+import { useHasFeature } from '@/hooks/useChains'
+import madProps from '@/utils/mad-props'
+import type { FEATURES } from '@/utils/chains'
+
+// TODO: Use with swaps/staking
+export function _FeatureWrapper({
+  children,
+  feature,
+  to,
```

Comment:
I would rename this prop to `fallbackRoute`.

Reply:
Renamed in c5dc7e745121a5eefdb5a14ba92986d21b08d224.